src/platform/qt/MemorySearch.cpp

```diff
@@ -125,9 +126,16 @@ bool MemorySearch::createParams(mCoreMemorySearchParams* params) {
 	}
 	if (m_ui.typeStr->isChecked()) {
 		params->type = mCORE_MEMORY_SEARCH_STRING;
-		m_string = m_ui.value->text().toLocal8Bit();
+		QString enc = m_ui.encoding->currentText();
+		if (enc == "ASCII") {
```

Comment:
ASCII and Latin-1/ISO-8859-1 are not the same thing. It's called ISO-8859-1 in the memory view. Please keep it consistent.